src/components/CurrencyLogo/index.tsx

```diff
@@ -41,7 +57,12 @@ export default function CurrencyLogo({
     if (!currency || currency.isNative) return []
 
     if (currency.isToken) {
-      const defaultUrls = currency.chainId === 1 ? [getTokenLogoURL(currency.address)] : []
+      const defaultUrls = [SupportedChainId.ARBITRUM_ONE, SupportedChainId.MAINNET, SupportedChainId.OPTIMISM].includes(
```

Comment:
would it be useful to move this check inside `getTokenLogoURL` so the caller doesn't need to be aware of the impl details?